Automattic/woocommerce-payments
# Update priority for Multi-Currency filters for prices and currency

Fixes #4938

#### Changes proposed in this Pull Request

* Change the priority for Multi-Currency to modify the price and currency for products.
* 900 was chosen as the priority to allow for additional filtering to happen if needed. This can actually be seen in one of the modified tests.

#### Testing instructions





The steps below can be done with the `develop` branch to see the bug happen, then with the PR branch to see the bug fixed.

* Make sure you have Multi-Currency enabled under Payments > Settings.
* Under WooCommerce > Settings > Multi-Currency, make sure you have at least one additional currency added, such as JPY. (I found it's easier to see the bug with JPY)
* If you have not added the Multi-Currency widget, this will need to be added.
  * Under Appearance > Widgets add the Currency Switcher Block to a sidebar on your site.
* Install WooCommerce Product Bundles. This can be found in the all plugins repo. See p1666095361999379-slack-C01B8KNUYSW
* Create a new product, and select Product Bundle as the type.
  * Set the product price as 125.
  * Under _Bundled products_ on the left hand side, add an existing product as part of the bundle.
  * There will be a _Priced Individually_ checkbox under the product added as part of the bundle, check it.
  * Set a % discount, such as 30.
  * Repeat the above steps again to add a second product to the bundle.
  * Save the product.
* View product and confirm the price gets converted on the product page when selecting a new currency.
* Add product to cart and view cart. (Screenshot shows cart widget in sidebar)
* Confirm the product price is still correctly converted in the cart.
* Move to checkout.
* Confirm the product price is still correctly converted on checkout.

Product settings:
![Markup 2022-10-18 at 10 18 28](https://user-images.githubusercontent.com/4634416/196443150-1b4b9ee3-91a2-4f78-8fd6-1645d2b97066.png)

Incorrect conversion:
![Markup 2022-10-18 at 10 19 29](https://user-images.githubusercontent.com/4634416/196443212-d9499d11-fe94-470b-b18d-df185bee7175.png)


-------------------


**Post merge**
